.codespellrc

```diff
@@ -0,0 +1,8 @@
+[codespell]
+skip = *.po,_build,_static,porting_guide*,collections,style_guide,
```

Comment:
I prefer the newline-separated INI list format. I think it's easier to read, but that's up to you :). At least, it'd be good if there were spaces between items in the comma-separated format.

```suggestion
skip = 
    *.po
    _static
    build
    collections
    porting_guide*
    style_guide
```

(I sorted the items in the process)